Wrappers/Python/ccpi/web_viewer/dev-environment.yml

```diff
@@ -1,11 +1,11 @@
 # run: conda env create --file environment.yml
-name: cilviewer_webapp
+name: cilviewer_webapp_trame2
 dependencies:
 - python==3.9
 - matplotlib # Optional for more colormaps
 - h5py
 - pip
 - pip:
   # Have to install Trame via pip due to unavailability on conda
-  - trame<2.0  # pin version for pip
+  - trame  # pin version for pip
```

Comment:
Should be handled in the other PR https://github.com/vais-ral/CILViewer/pull/262